### PR TITLE
filters/diag: add repeatContentHex and wrapContentHex

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

File: docs/reference/filters.md [hunk 1]
```diff
@@ -486,6 +506,33 @@ Examples:
...
+
+```
+// 1G of gzip-compressed text
+*
+-> setResponseHeader("Content-Encoding", "gzip")
+-> wrapContentHex(
+  "1f8b08000000000004ffecd6b10d00200804c05598c5b80a852d0ee422762ce61c2657d212f8bf9915bb6f9f8c51b9c26c1feec13fc80379a80ff4210ff0000ff0000ff0000ff0000ff0000ff0000ff0000ff0000ff0000ff0000ff0000ffce781070000ffffecd6810c000000c0207feb737c8ba2f8cd6f7ef39bdffce637bf",
+  "7dc0077cc0077cc0077cc0077cc0077cc0077cc0077cc0077cc0077cc0077cc0077ce0ff81000000ffffecd6810000000080207feb418ea278ce739ef39ce73ce739cf7de0f581000000ffff010000ffff5216994600ca9a3b"
+)
+-> repeatContentHex("7dc0077cc0077cc0077cc0077cc0077cc0077cc0077cc0077cc0077cc0077cc0077ce0ff81000000ffffecd6810c000000c0207feb737c8ba278ce739ef39ce73ce739cf", 8300624)
```
Comment (AlexanderYastrebov) [inline docs/reference/filters.md:532]: One may use https://github.com/AlexanderYastrebov/unrepeat to find these values, not sure if we should link it in the docs.

Comment (szuecs) [reply docs/reference/filters.md:532]: If you think it's valuable, of course link it.

Comment (AlexanderYastrebov) [reply docs/reference/filters.md:532]: Ok, added a link

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**